src/awfy.js

```diff
@@ -206,6 +105,30 @@ const DESKTOP_APPS = {
   },
 };
 
+const WASM_APPS = {
+  ...DESKTOP_APPS,
+  firefox: {
```

Comment:
In DESTOP_APPS -> DESKTOP_FIREFOX_APPS there is already a `firefox` app:

```
firefox: {
    name: 'firefox',
    label: 'Firefox',
    color: PALETTE.orange,
    project: ALT_PROJECT,
  },
```

I'm didn't see any error related to this, but it may be an issue

Reply:
I overwrite to adjust the label.